ios/Classes/SwiftMparticleFlutterSdkPlugin.swift

```diff
@@ -77,7 +77,7 @@ public class SwiftMparticleFlutterSdkPlugin: NSObject, FlutterPlugin {
         } else {
             print("Incorrect argument for \(call.method) iOS method")
         }
-    case "setATTStatus":
+    case "setAttStatus":
```

Comment:
maybe the casing should be consistent throughout the Dart API down through the Swift Layer? Looks like Swift does `setATTstatus` also.  can you change the case here back to `setATTStatus` as well as line 161 in `lib/mparticle_flutter_sdk.dart`?